lib/queries/useInstanceDateTypes/useInstanceDateTypes.js

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+import {
+  CQL_FIND_ALL,
+  LIMIT_MAX,
+} from '../../constants';
+
+export const useInstanceDateTypes = () => {
+  const ky = useOkapiKy();
+  const [namespace] = useNamespace({ key: 'instance-date-types' });
+
+  const query = useQuery({
+    queryKey: [namespace, 'instance-date-types'],
+    queryFn: () => ky.get(`instance-date-types?limit=${LIMIT_MAX}&query=${CQL_FIND_ALL}`).json(),
+  });

```

Comment:
Let's add `tenantId` to support tenant switching functionality in the `receiving` app (https://github.com/folio-org/stripes-inventory-components/pull/66).